web/src/page_params.ts

```diff
@@ -19,6 +20,7 @@ export const page_params: {
     is_moderator: boolean;
     is_owner: boolean;
     is_spectator: boolean;
+    max_avatar_file_size_mib: number | undefined;
```

Comment:
Hmm, I didn't expect this to be optional. Any specific reason for that?

It seems to be inspired by needing a default for `build_direct_upload_widget`, which was added in 519e8471c9829cddb52de8022d482e4288c63502, but I don't see how page param will have `max_avatar_file_size_mib` missing

Reply:
@PIG208 Yeah, it looks `max_avatar_file_size_mib` will always be present in `page_params` I tracked down the code path for `build_page_params_for_home_page_load` and I can confirm that this is the case.

Also looking at all the callers of `upload_widget.build_direct_upload_widget` it seems that this default value is actually not required here and `max_file_upload_size` will always be defined. So shall we remove this default variable?

Reply:
Yeah, that seems correct -- we use `0` to mean uploads are disabled.